### ISSUE TITLE
phpbbauth blocking search engines?

### ISSUE THREAD

**Issue by tgbholidayboy** [multidimension-al/phpbbauth#3]
Hi,

I've installed your add-in and everything works great!

https://www.thegreenbutton.tv/wiki/

I have an issue with all search engines getting internal server 500 errors when it's active though.

I've confirmed this by removing the following entry in localsettings.php:   require_once "$IP/extensions/Phpbbauth/PhpbbAuth.php"

Search engines get an OK 200 response when the add-in is not in use.

Does anybody else see this issue, or have any ideas how to work around it please?

**Comment by ajquick** [comment]
Looking into this. I just tested it on your site and switched my user-agent to "Googlebot".

I get the following error: 

> [W5hddLx5KZcAAFQOmN8AAAAm] 2018-09-12 00:27:33: Fatal exception of type "InvalidArgumentException"

phpBB handles Bots kind of like a logged in user, so I will have to investigate this and get back to you. It is undeniably very important to get it fixed!

**Comment by tgbholidayboy** [comment]
Thanks for taking the time to look at it, I'll try the new version and report back.

I recently upgraded to phpbb 3.2.2, I was previoulsy on a 3.0x version and used the older auth_phpbb add-in with MW 1.2x

I did wonder after posting if it may actually be the auh_remoteuser add-in that was the cause and found item 35 in this list:

https://www.mediawiki.org/wiki/Extension_talk:Auth_remoteuser

I tried changing minimalpassword lengths etc but still had the same issue, the second option no longer seems to be an option as the code has been refactored and the file mentioned no longer exists.

The issue was definitely present before I installed the cleantalk add-in, I'll uninstall it temporarily while testing the new version just to be sure.

Thanks again.
